tools/ds-identify

```diff
@@ -281,6 +280,49 @@ read_fs_info() {
     DI_ISO9660_DEVS="${isodevs#,}"
 }
 
+read_geom() {
+    local oifs="$IFS" line="" delim=","
+    local ret=0 out="" labels="" dev="" label="" ftype="" isodevs="" uuids=""
+    out=$(geom label status -as) || {
+        ret=$?
+        error "failed running [$ret]: geom label status -as"
+        DI_FS_LABELS="$UNAVAILABLE:error"
+        DI_ISO9660_DEVS="$UNAVAILABLE:error"
+        return $ret
+    }
+
+    # shellcheck disable=2086
+    { IFS="$CR"; set -- $out; IFS="$oifs"; }
+
+    for line in "$@"; do
+        set -- $line
+        provider=$1
+        ftype="${provider%/*}"
+        label="${provider#*/}"
+	dev=$3
```

Comment:
white space.

Reply:
thanks!

(this was written in an unconfigured nvim on FreeBSD, and then just transferred over…)